src/lucky/json_body_parser.cr

```diff
@@ -0,0 +1,21 @@
+# :nodoc:
+class Lucky::JsonBodyParser
+  getter request : HTTP::Request
+
+  def initialize(@request)
+  end
+
+  def parsed_json : JSON::Any
+    if body.blank?
+      JSON::Any.new({} of String => String)
```

Comment:
I think this needs needs to be `{} of String => JSON::Any`

Reply:
Yeah I got to trigger happy and forgot to run specs. Oops :)